File: Assignment No.37/Program3.java
/*Problem Statement :Write a Program which accept one number and whether that 7 bit,21 bit,28 bit,15 bit is on or off.
Input : 10 
Output : off
*/
import java.util.*;

class Program3
{
	public static boolean ChkBit(int iNo)
	{
		int iMask=0x08104040,iResult=0;

		iResult=iNo & iMask;
		if(iResult == iMask)
		{
			return true;
		}
		else
		{
			return false;
		}
			 
	}
	public static void main(String args[])
	{
		int iNo = 0;
		Boolean bret=false;

		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the Number :");
		iNo = sobj.nextInt();
		
		bret = ChkBit(iNo);
		if(bret == true)
		{
			System.out.println("7,21,28,15 bit is on");
		}
		else
		{
			System.out.println("7,21,28,15 bit is off");
		}
		
	}
}	
		

File: Assignment No 4/Program3.c
//Write a Program which accept number from user and display all its non factors .
//Input : 12
//Output : 5 7 8 9 10 11
//Input : 13
//Output : 2 3 4 5 6 7 8 9 10 11 12
#include<stdio.h>

void FactRev(int iNo)
{
	int iFact=0;

	for(iFact=1;iFact<=iNo;iFact++)
	{
		if((iNo%iFact)!=0)
		{
			printf("%d\t",iFact);
		}
	}
}
int main()
{
	int iValue =0;
	int iRet=0;

	printf("Enter the number :\n");
	scanf("%d",&iValue);

	FactRev(iValue);
	
	return 0;
}